ui/js/constants.js

```diff
@@ -179,24 +179,24 @@ export const thSimplePlatforms = [
 
 export const thFailureResults = ['testfailed', 'busted', 'exception'];
 
-export const thAllResultStates = [
-  'success',
+export const thAllResultStatuses = [
   'testfailed',
   'busted',
   'exception',
+  'success',
```

Comment:
reordering these so we don't have to munge them to show the menu in the right order.